cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -177,6 +177,9 @@ rmm::device_buffer make_elements(InputIterator begin, InputIterator end)
     elements.data(), size * sizeof(ElementTo), cudf::test::get_default_stream()};
 }
 
+// The two signatures below are identical to the above overload apart from
+// SFINAE so doxygen sees it as a duplicate.
+//! @cond Doxygen_Suppress
```

Comment:
will `@copydoc` work without warnings/errors?
Add an extra line below in addition to copydoc mentioning that this is specialization for that type (fixed_point, non-fixed_point).

Reply:
No, that didn't work for me. The problem is that the function signature is the same so doxygen sees two identical functions. It doesn't seem to understand the template specialization via SFINAE, only explicitly template specialization.